### PR TITLE
Fix start_time and create answer_time accessor

### LINKED ISSUE #601: Consistent handling for call start time
From `lib/adhearsion/call.rb`:

``` Ruby
    # @return [Time] the time at which the call began. For inbound calls this is the time at which the call was offered to Adhearsion. For outbound calls it is the time at which the remote party answered.
    attr_reader :start_time
```

This is inconsistent. I think this should be:
- For inbound calls, the time the call was offered to Adhearsion
- For outbound calls, the time at which the call was dialed

We can probably also add an `:answer_time` attribute which tracks the time at which the call was answered.

### PR DESCRIPTION
**Pull request by bklang**
Closes #601

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
   * Bugfix: Handle more AMI error responses which mean the channel is not found
   * Bugfix: Reporting statistics should not deadlock call processing
   * Bugfix: Sending the `USR2` signal to an Adhearsion application now prints thread stack traces to STDERR.
+  * Change: Make Adhearsion::Call#start_time now consistently tracks the call start time for both incoming and outgoing calls
```
Comment (benlangfeld) [inline CHANGELOG.md]: Buggy grammar.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+  * Feature: Add Adhearsion::Call#answer_time to track the time at which the call was answered
```
Comment (benlangfeld) [inline CHANGELOG.md]: Please surround class names, method names, etc with code fences.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
...
+  * Feature: Send Adhearsion::Event::Answered events when inbound calls are answered (formerly was only sent when outbound calls were answered)
```
Comment (benlangfeld) [inline CHANGELOG.md]: This is the responsibility of the server and should also go in the Rayo spec

**Comment by bklang** [comment]
@benlangfeld review?

**Comment by benlangfeld** [comment]
LGTM

**Pull request merged**